README.md

```diff
@@ -35,3 +35,9 @@ enter the room by pointing to the arrow button
 use the ... menu
 
 select the correct BBB room and enter the room
+
+# Troubleshooting
+- before installing, make sure your BBB is running correctly
+- sameSite Cookies issue in modern chrome/chromium is to be expected. Always use the same tld.org
```

Comment:
What do you mean with "same site cookies issue"?

Do you have a screenshot from your error message?

Reply:
see https://github.com/ramezrafla/spreed-bigbluebutton

Reply:
What should I see? The architecture is completely different and we should not suffer from any cross domain issues.

Reply:
will retest and eventually remove this line, sorry

Reply:
> sorry

no problem

Reply:
retested & removed that line - we're not having this issue. Sorry.